### PR TITLE
GraphQL Response has unexpected field "specification" in Micronaut 4.0.4

### PR DESCRIPTION
**Pull request by guillermocalvo**

### CODE REVIEW

**Review by sdelamo — APPROVED** [review]

File: graphql/src/main/java/io/micronaut/configuration/graphql/GraphQLResponseBody.java [hunk 1]
```diff
@@ -51,7 +51,7 @@ public GraphQLResponseBody(Map<String, Object> specification) {
      * @return the GraphQL response body data
      */
     @JsonAnyGetter
-    @JsonInclude
+    @JsonIgnore
```
Comment (sdelamo) [inline graphql/src/main/java/io/micronaut/configuration/graphql/GraphQLResponseBody.java:54]: why and when was `@JsonInclude`added?

File: graphql/src/main/java/io/micronaut/configuration/graphql/GraphQLResponseBody.java [hunk 2]
```diff
@@ -51,7 +51,7 @@ public GraphQLResponseBody(Map<String, Object> specification) {
      * @return the GraphQL response body data
      */
     @JsonAnyGetter
```
Comment (sdelamo) [inline graphql/src/main/java/io/micronaut/configuration/graphql/GraphQLResponseBody.java:53]: Should we remove `@JsonAnyGetter` if we are going to annotate with `@JsonIgnore`?

**Comment by yawkat** [comment]
this sounds like it'd be fixed by https://github.com/micronaut-projects/micronaut-core/pull/9872

File: graphql/src/main/java/io/micronaut/configuration/graphql/GraphQLResponseBody.java (hunk 1 above)
Comment (guillermocalvo) [reply graphql/src/main/java/io/micronaut/configuration/graphql/GraphQLResponseBody.java:54]: According to git blame, it's been there from the very beginning: https://github.com/micronaut-projects/micronaut-graphql/commit/96589153d7f0c4493b7157abeb291c8ffc050d08#diff-5185b30fe44c67e5506bf2d1b0b8cf5794ff5433a7dbfdc26ffa79f95a294678R39

File: graphql/src/main/java/io/micronaut/configuration/graphql/GraphQLResponseBody.java (hunk 2 above)
Comment (guillermocalvo) [reply graphql/src/main/java/io/micronaut/configuration/graphql/GraphQLResponseBody.java:53]: No, I don't think so. Those two annotations serve different purposes.

**Comment by yawkat** [comment]
we should keep it as-is if it's fixed already in core

**Review by sdelamo — APPROVED** [review]

**Pull request merged**